src/wallet/rpcwallet.cpp

```diff
@@ -2598,8 +2599,21 @@ static UniValue createwallet(const JSONRPCRequest& request)
         flags |= WALLET_FLAG_DISABLE_PRIVATE_KEYS;
     }
 
+    bool create_blank = false;
     if (!request.params[2].isNull() && request.params[2].get_bool()) {
         flags |= WALLET_FLAG_BLANK_WALLET;
+        create_blank = true;
+    }
+    SecureString passphrase;
```

Comment:
nit: move to a shared function so it's more clear this is identical to `encryptwallet`

Reply:
It's not identical. There isn't enough shared here to warrant that IMO